toot/cli/tui.py

```diff
@@ -24,12 +24,18 @@
     help=f"""Number of colors to use, one of {COLOR_OPTIONS}, defaults to 16 if
              using --color, and 1 if using --no-color."""
 )
+@click.option(
+    "-V", "--default-visibility",
```

Comment:
Any reason for having an uppercase `-V` instead of lowercase?

Reply:
i used `-v` originally, but this is often used for a 'verbose' option by convention, so i went with `-V` instead to avoid a possible future conflict.  i don't have any particular preference though, i can switch it to `-v` if you like.

Reply:
I doubt we'll have a verbose option for tui so please change it to `-v`. This will match the shorthand for `--visiblity` on `toot post`.

Reply:
changed to -v